src/main/resources/Dockerfile

```diff
@@ -15,12 +11,7 @@ ENV TARANTOOL_INSTANCES_FILE=$TARANTOOL_INSTANCES_FILE
 RUN curl -L https://tarantool.io/installer.sh | VER=$TARANTOOL_VERSION /bin/bash -s -- --repo-only && \
     yum -y install cmake make gcc gcc-c++ git unzip tarantool tarantool-devel cartridge-cli && \
     yum clean all
-RUN groupadd -g $TARANTOOL_SERVER_GID $TARANTOOL_SERVER_GROUP && \
-    useradd -u $TARANTOOL_SERVER_UID -g $TARANTOOL_SERVER_GID -m -s /bin/bash $TARANTOOL_SERVER_USER \
-    || true
-USER $TARANTOOL_SERVER_USER:$TARANTOOL_SERVER_GROUP
-RUN cartridge version
 
-FROM tarantool-base AS cartridge-base
```

Comment:
Removing this breaks the layer caching. Even if it currently doesn't work in Testcontainers (shame on them), the new approach with plain docker-java can make this useful, since most of the build time is wasted on installing the libs.

Reply:
Ok, reverted the change